### PR TITLE
remove old gazebo settings

### PR DESCRIPTION
**Pull request by YoheiKakiuchi**
removing old gazebo settings, they does not match latest gazebo.

### CODE REVIEW

**Comment by isucan** [comment]
@hsu @nkoenig ?

File: collada_urdf/src/collada_to_urdf.cpp [hunk 1]
```diff
@@ -508,6 +508,7 @@ void printTreeXML(boost::shared_ptr<const Link> link, string name, string file)
   os << "<?xml version=\"1.0\"?>" << endl;
   os << "<robot name=\"" << name << "\"" << endl;
   os << "       xmlns:xi=\"http://www.w3.org/2001/XInclude\"" << endl;
+#if GAZEBO_1_0 // old style, do not match gazebo 4.1
```
Comment (nkoenig) [inline collada_urdf/src/collada_to_urdf.cpp:511]: You can remove the old style. The probability of someone using gazebo 1.0 with indigo is essentially zero.

**Comment by jackieokay** [comment]
@YoheiKakiuchi: it sounds like based on @nkoenig's feedback, we can just remove the part that you #ifdef out in this pull request for Indigo, as well as the other references to GAZEBO_1_0.

All links under `playerstage.sourceforge.net/gazebo` in this file are broken anyway. I'm not sure if we want to purge them in Hydro, or if we still need them for some reason.

**Comment by clalancette** [comment]
Based on the comment from Nate, I ended up pushing a patch that just removed all of the GAZEBO_1_0 sections of code in this file.  I'm going to close this out now; thanks for the contribution.

**Pull request closed without merging**